### PR TITLE
Add client for NERC OCP Prod

### PR DESCRIPTION
**Pull request by knikolla**

### CODE REVIEW

**Comment by joachimweyl** [comment]
@jtriley & @culbert this is a rather short PR, will a review of this fit before you are done with Swift?

File: k8s/overlays/nerc-shift-1/clients/ocp-prod.yaml [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        claim.name: clientId
+        id.token.claim: "true"
+        jsonType.label: String
+        user.session.note: clientId
+      name: Client ID
+      protocol: openid-connect
+      protocolMapper: oidc-usersessionmodel-note-mapper
+    publicClient: false
+    redirectUris:
+    - https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/*
```
Comment (jtriley) [inline k8s/overlays/nerc-shift-1/clients/ocp-prod.yaml]: Might also want to include the public version of this: `https://console-openshift-console.apps.shift.nerc.mghpcc.org/*`. Does the oauth route also need to be included here (for both for internal and public views)?

**Review by jtriley — CHANGES_REQUESTED** [review]

File: k8s/overlays/nerc-shift-1/clients/ocp-prod.yaml (hunk 1 above)
Comment (knikolla) [reply k8s/overlays/nerc-shift-1/clients/ocp-prod.yaml]: Added the public version and the oauth route.

**Review by jtriley — APPROVED** [review]

**Review by culbert — APPROVED** [review]
LGTM.

**Pull request merged**